projects/ngx-prx-styleguide/src/lib/hal/remote/halremote.spec.ts

```diff
@@ -20,16 +21,9 @@ describe('HalRemote', () => {
     httpClient = TestBed.get(HttpClient);
 
     link = { href: '/foobar' };
-    token = new ReplaySubject<string>(1);
-    token.next('thetoken');
-    fakeAuth = {
-      token,
-      refreshToken: () => {
-        token.next('nexttoken');
-        return token;
-      }
-    };
-    remote = new HalRemote('http://thehost', httpClient, fakeAuth, 10);
+    auth = new AuthService();
```

Comment:
I switched the mock auth service out for the real one ... just to make sure it's using the same token/retry logic.